cognite/client/data_classes/datapoints.py

```diff
@@ -295,7 +316,8 @@ def to_payload_item(self) -> _DatapointsPayloadItem:
             payload["ignoreBadDataPoints"] = self.ignore_bad_datapoints
         if self.treat_uncertain_as_bad is False:
             payload["treatUncertainAsBad"] = self.treat_uncertain_as_bad
-
+        if self.timezone:
+            payload["timeZone"] = self.timezone
```

Comment:
I could have made an `api_friendly_timezone` parameter instead of overwriting the existing one, but then the fetching logic must use that one instead of the logical one (`timezone`), so I wouch to keep it. Pleeeenty of tests that cover all possible inputs.